packages/web3torrent/src/library/paid-streaming-extension.ts

```diff
@@ -14,13 +14,17 @@ export abstract class PaidStreamingExtension implements Extension {
   protected wire: PaidStreamingWire;
   protected messageBus: EventEmitter;
   protected pseId: string = '';
+  protected pseOutcomeAddress: string = '';
 
   get name(): 'paidStreamingExtension' {
     return 'paidStreamingExtension';
   }
 
   peerAccount?: string;
+  peerOutcomeAddress?: string;
```

Comment:
Why do we need `?` if there is a default value  of `''`?

Reply:
If the default value that you are referring to is for the `pseOutcomeAddress`, which is set at the wire creation, not on the extended handshake.
I tried to respect the way the props were being handled previously.
The idea is that both `peerAccount` and `peerOutcomeAddress` are initially undefined because they are not set at the wire setup, while `pseId` and `pseOutcomeAddress` are defaulted to '', because they are set at the wire setup.